README.md

```diff
@@ -1,27 +1,27 @@
 # Info
-Library is provided by Loupe  
+Package is provided by Loupe  
 https://loupe.team  
 info@loupe.team  
 1-800-240-7042  
 
 # Description
-This repo provides packages that allow basic control of B&R Mapp Motion servo axes.  
-
+This repo provides packages that allow basic control of B&R servo axes.  
 ### Program Package
-This program package will install the Servo B&R package that will contain the AxisBasic program for a Automation Studio project and all its dependencies that supports Mapp Motion.
-The AxisBasic program will call the AxisBasic_Cyclic function block which handles much of the basic axis control.
+This program package will install the Servo B&R package that will contain the ServoMgr program for a Automation Studio project and all its dependencies.
+The ServoMgrInit program will perform simple initialization of parameters if needed.
+The ServoMgrCyclic program will call the AxisBasicFn_Cyclic function block which handles much of the basic axis control.
+The cyclic program also calls the axisErrorCollector function which will support axis error handling.
 
 ### Library Package
-This library package allows access to the axis basic function blocks.
+This library package allows access to the servo basic function blocks.
 
 # Installation
-To install using the Loupe Package Manager (LPM), in an initialized Automation Studio project directory run `lpm install mappmotionprog` for mappMotion or run `lpm install acp10motionprog` for ACP10. Note that this will also pull in the library package as a dependency.  
-If you only want to install the library package, run `lpm install axislib`.   
-For more information about LPM, see https://loupeteam.github.io/LoupeDocs/tools/lpm.html
+To install using the Loupe Package Manager (LPM), in an initialized Automation Studio project directory run `lpm install acp10motionprog`.  Note that this will also pull in the library package as a dependency.  
```

Comment:
How would you feel about changing this to reference `mappmotionprog` instead, since this will probably be our default choice for future applications?

Reply:
Yeah lets do it 💪